packages/cli/src/lib/bundler/server.ts

```diff
@@ -199,7 +203,26 @@ DEPRECATION WARNING: React Router Beta is deprecated and support for it will be
       root: paths.targetPath,
     });
   } else {
-    const compiler = webpack(config);
+    const authPaths = await resolveOptionalBundlingPaths({
+      entry: 'src/auth',
+      dist: 'dist/auth',
+    });
+    if (authPaths) {
+      console.log(
+        chalk.yellow(
+          `⚠️  WARNING: The app /auth entry point is an experimental feature that may receive immediate breaking changes.`,
+        ),
+      );
+    }
+    const compiler = authPaths
+      ? webpack([
```

Comment:
When doing this as an array, will webpack make shared chunks and thus the _contents_ of the auth path will still equally be at risk of serving parts of pwebsite source with private plugins etc? I.e., is this separate endpoint not so much looking to solve bundle content secrecy problems as such? Or does it indeed keep them entirely separate as long as the auth part doesn't explicitly import those secret things

Reply:
It ends up as a complete standalone bundle, with any common code duplicated across the two entry points. The auth bundle with only contain the code that is reachable from the auth entry point, and the auto discovered modules are not included.